src/core/Table/Table.test.tsx

```diff
@@ -406,13 +406,23 @@ it('should not show sorting icon if sorting is disabled', () => {
 
 it('should not show sort icon if data is loading', () => {
```

Comment:
Amm, test name now is wrong. Maybe say `if there is no data` isntead

Reply:
No, it is not 😀 `data` is required field therefore passing empty array and `isLoading: true`